src/linear_algebra/basic.lean

```diff
@@ -2462,7 +2462,7 @@ variables (K) (M)
 open _root_.linear_map
 
 /-- Multiplying by a nonzero element `a` of the field `K` is a linear equivalence. -/
-def smul_of_ne_zero (a : K) (ha : a ≠ 0) : M ≃ₗ[K] M :=
+@[simps] def smul_of_ne_zero (a : K) (ha : a ≠ 0) : M ≃ₗ[K] M :=
```

Comment:
This strictly speaking belongs to the next PR (if it even matters ...)

Reply:
I don't think it even matters, but I restored it anyway!